pandas/core/arrays/period.py

```diff
@@ -52,13 +65,17 @@ def _period_array_cmp(cls, op):
 
     def wrapper(self, other):
         op = getattr(self._ndarray_values, opname)
+        if isinstance(other, (ABCSeries, ABCIndexClass)):
+            # TODO: return NotImplemented?
+            other = other.values
```

Comment:
That's in https://github.com/pandas-dev/pandas/pull/23155 (not merged yet, but I think ready to go).

We'll need to re-implement / adjust the test, since it uses `EA.__add__(Series[EA])`, which isn't defined for PeriodArray. We can do `__sub__`.

Reply:
I had to add an xfail for `PeriodArray == Series[PeriodArray]`. We don't return NotImplemented for comparison yet. But I think changing ops is out of scope for this PR.